helix-term/src/main.rs

```diff
@@ -134,6 +137,19 @@ FLAGS:
         }
     };
 
+    if !ignored_keys.is_empty() {
+        let keys = ignored_keys.into_iter().collect::<Vec<_>>().join(", ");
+        eprintln!("Ignored keys in config: {}", keys);
+        set_config_error();
```

Comment:
I think if we do it this way with global static variable, later for reloading config, we need to handle this, I wonder if it would be better if we can just propagate the error upwards.

Reply:
The problem is that serde is it's own thing, so it's not that easy.
Anyway, I'm still waiting for the answer in the discussion as it seems some people are against the idea of this PR.

Maybe the way this is handled will change after we discuss the design.